owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/shares/RemoteShare.kt

```diff
@@ -37,124 +33,27 @@ import java.io.Serializable
  * @author David A. Velasco
  * @author David González Verdugo
  */
-class RemoteShare : Parcelable, Serializable {
-    var id: Long = 0
-    var shareWith: String = ""
-    var path: String = ""
-    var token: String = ""
-    var sharedWithDisplayName: String = ""
-    var sharedWithAdditionalInfo: String = ""
-    var name: String = ""
-    var shareLink: String = ""
-    var fileSource: Long = 0
-    var itemSource: Long = 0
-    var shareType: ShareType? = null
-    var permissions: Int = DEFAULT_PERMISSION
-    var sharedDate: Long = INIT_SHARED_DATE
-    var expirationDate: Long = INIT_EXPIRATION_DATE_IN_MILLIS
-    var isFolder: Boolean = path.endsWith(FileUtils.PATH_SEPARATOR)
-    var userId: Long = 0
-
+data class RemoteShare(
```

Comment:
This is class is neither parcelable nor serializable, are you sure?

Reply:
Yes, is no longer used, OCShare in the domain will keep being parcelable